### PR TITLE
add docker-kubectl-helm dockerfile

### PR DESCRIPTION
**Pull request by WellsCui**
This image is built to support running tests with Kubernetes cluster in CI environment. It provides utilities such as docker, kubectl and helm. See the `Dockerfile` in this folder for details.

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Hi, and thanks for contributing!

Note: If you are part of Qlik R&D, you can ignore this process (but let us know in a comment below!)

Before we can handle your pull request we need you to sign our contributor license agreement.
This is a one-time thing and enables you to contribute without hassle to any of the qlik-oss repositories in the future.

And it only takes a few minutes! Here are some useful links to get you started:

* [Sign the CLA](https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sign-the-cla)
* [Rationale behind the CLA](https://github.com/qlik-oss/open-source/blob/master/cla-rationale.md)
* [Qlik R&D Open Source repository](https://github.com/qlik-oss/open-source#open-source-at-qlik-rd)
* [Official Slack team](http://branch.qlik.com/slack)

Having troubles? Feel free to reach out to us either here or on Slack!

**Comment by peol** [comment]
CLA OK 👍

File: docker-kubectl-helm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+RUN apk add --update ca-certificates \
+ && apk add --no-cache curl make lz4 bash socat \
+ && apk add --update -t deps \
+ && apk add --update gettext tar gzip \
+ && curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl \
+ && chmod +x ./kubectl \
+ && mv ./kubectl /usr/local/bin/kubectl 
+
+ Run curl -L https://storage.googleapis.com/kubernetes-helm/${HELM_FILENAME} | tar xz && mv linux-amd64/helm /bin/helm && rm -rf linux-amd64 \
```
Comment (hairyhenderson) [inline docker-kubectl-helm/Dockerfile]:
```suggestion
RUN curl -L https://storage.googleapis.com/kubernetes-helm/${HELM_FILENAME} | tar xz && mv linux-amd64/helm /bin/helm && rm -rf linux-amd64 \
```

File: docker-kubectl-helm/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ && apk del --purge deps \
+ && rm /var/cache/apk/*
```
Comment (hairyhenderson) [inline docker-kubectl-helm/Dockerfile]:
```suggestion
 && rm /var/cache/apk/*
```

Comment (hairyhenderson) [reply docker-kubectl-helm/Dockerfile]: (adding a newline)

**Comment by hairyhenderson** [comment]
@WellsCui I wonder if adding `kubectl` and `helm` to one of the other build images (like `tiny-build` and `go-build`) would be a good idea instead?

**Comment by WellsCui** [comment]
> CLA OK 👍

Thanks!

**Pull request merged**